### ISSUE TITLE
[Feature Request] Instructions for bare-metal/LXC/VM installation without Docker

### ISSUE THREAD

**Issue by valantur** [crocodilestick/Calibre-Web-Automated#122]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
It would be great to have instructions on how to install CWA without Docker.

**Describe alternatives you've considered**
Clonning the project and running the python app from an LXC container w/o Docker.

**Additional context**
I believe a lot of people who for whatever reason can't or don't want to run Docker would appreciate having an alternative way to run this. I'll try to do it myself in the next couple of days by reading the dockerfile, but some people don't know how to do this and would find it easier to follow step-by-step instructions.

**Comment by PeteCreation** [comment]
Having to learn Docker on top of LXC and Proxmox has been a strain. I've built so many VMs and containers now and still not got this working so a very simple set of instructions would be really helpful.

**Comment by jmedemar** [comment]
Yes, using this in a LXC would be ideal, since I already have calibre-web running in one. A set of instructions to install this into its own container would be appreciated.

**Comment by Tooelite** [comment]
Just to update the issue there is a new script that installs a Proxmox LXC based on the original CWA Docker repo:

https://github.com/vhsdream/calibre-web-automated-lxc

**Comment by diginfotek** [comment]
This would be great. I've tried Tooelite's script but he stopped maintaining it and it currently fails, so no luck there.

I managed to get Calibre-web on an LXC with their manual install instructions, but there is no chance to make it work with CWA. Having the option to run outside Docker would be very much appreciated.
